org.civicrm.reqangular/test/services/notification_test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-env amd, jasmine */
+
+(function (CRM) {
+  define([
+    'common/angular',
+    'common/angularMocks',
+    'common/services/notification'
+  ], function () {
+    'use strict';
+
+    describe('notification', function () {
+      var title, body, options, notification;
+
+      beforeEach(module('common.services'));
+      beforeEach(inject(function (_notification_) {
+        notification = _notification_;
+        title = 'Sample title';
+        body = 'Sample Body text';
+        options = {
+          unique: true,
+          expires: 0
+        };
+
+        spyOn(CRM, 'alert').and.returnValue({});
+      }));
+
+      describe('alert()', function () {
+        it('calls CRM.alert to display alert notification', function () {
+          notification.alert(title, body, options);
```

Comment:
Call the function in beforeEach, it is not supposed to have the setup for the test. Check other places too.